index.css

```diff
@@ -0,0 +1,189 @@
+.img-responsive {
+  display: block;
+  width: 100%;
+  margin-top: 1rem;
+  height: auto;
+  object-fit: cover;
+}
+
+.hero-section {
+  margin-top: 4.2rem;
+  padding-left: 4rem;
+  padding-right: 4rem;
+  background-size: cover;
+}
+
+.h2 {
+  font-size: 2rem;
+  font-weight: 800;
+}
+
+.fw-400 {
+  padding-left: 2px;
+  font-weight: 400;
+}
+
+.hero-text {
+  position: absolute;
+  top: 20%;
+  right: 15%;
```

Comment:
instead of using % can use rems, because % uses parent's reference which can be a scaling issue later on